### PR TITLE
Create compact page title styles

### PR DESCRIPTION
**Pull request by ohskylab**
TP: https://rockabox.tpondemand.com/entity/9786

Add compact `rb-page-title` styles and demo. Needs Angularifying and demo fixing.

Note that utility classes on `.PageTitle-heading`, `.PageTitle-sub` and `.PageTitle-navigation` descendents need switching between standard and compact variants.

### CODE REVIEW

File: src/rb-page-title/demo/demo.tpl.html [hunk 1]
```diff
@@ -34,4 +37,33 @@
...
+                        <h1 class="u-textHeading2">
+                            Title of section
+                        </h1>
+                    </div>
+                </div>
+                <p class="PageTitle-sub u-textDefault">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                </p>
+            </div>
+            <div class="PageTitle-button">
```
Comment (chrisvxd) [inline src/rb-page-title/demo/demo.tpl.html]: Needs updating now

**Comment by chrisvxd** [comment]
FYI task branch names generally take the form `owner/id-description`, and story branches `story/id-description`. It was documented somewhere but I think it's lost in the ether now... I'll make a note to reintroduce that.

File: src/rb-page-title/demo/demo.tpl.html (hunk 1 above)
Comment (ohskylab) [inline src/rb-page-title/demo/demo.tpl.html]: Can you clarify?

Comment (chrisvxd) [inline src/rb-page-title/demo/demo.tpl.html]: `PageTitle-action` instead of `-button`

**Pull request merged**